lib/ClimaLandSimulations/src/Fluxnet/fluxnet_utilities/met_drivers_fluxnet.jl

```diff
@@ -198,26 +196,28 @@ function setup_drivers(site_ID)
     LAI_dt = Second(MODIS_LAI[2, 1] - MODIS_LAI[1, 1]).value
     LAI_seconds = FT.(0:LAI_dt:((length(MODIS_LAI[:, 1]) - 1) * LAI_dt))
 
-    # LAI function for radiative transfer
+    # LAI for radiative transfer
     LAIfunction = TimeVaryingInput(LAI_seconds, FT.(MODIS_LAI[:, 2]); context)
 
     # Necessary inputs from LAI Data
     # Note that f_root_to_shoot, capacity, and h_leaf are site-specific parameters
     # defined in the parameters file for each site
     maxLAI = FT(maximum(MODIS_LAI[:, 2]))
-    RAI = maxLAI * f_root_to_shoot
-    plant_ν = capacity / (maxLAI * h_leaf) / FT(1000)
+    RAI = maxLAI * params.plant_hydraulics.f_root_to_shoot
+    plant_ν =
+        params.plant_hydraulics.capacity / (maxLAI * site_setup_out.h_leaf) /
+        FT(1000)
 
     return (
-        LOCAL_DATETIME,
-        atmos_co2,
-        DATA_DT,
-        drivers,
-        atmos,
-        radiation,
-        LAIfunction,
-        maxLAI,
-        RAI,
-        plant_ν,
+        LOCAL_DATETIME = LOCAL_DATETIME,
+        atmos_co2 = atmos_co2,
+        DATA_DT = DATA_DT,
+        drivers = drivers,
+        atmos = atmos,
+        radiation = radiation,
+        LAIfunction = LAIfunction,
```

Comment:
will this always be a function, or should we rename this more generally to allow for other types of LAI input?

Reply:
good point! for global runs it will be read from a netcdf file. later PR ok!

Reply:
I'll leave this for later!